File: AngularLovelyjubbly.Data.Sql.Models/Yardage.cs
namespace AngularLovelyjubbly.Data.Sql.Models
{
    public partial class Yardage
    {
        public Yardage()
        {
        }

        public int YardageId { get; set; }
        //foreign key
        public int TournamentId { get; set; }
        //navigation property
        public virtual Tournament Tournament { get; set; }
        //foreign key
        public int TeamId { get; set; }
        //navigation property
        public virtual Team Team { get; set; }
        public int OffensePassingYards { get; set; }
        public int OffenseRushingYards { get; set; }
        public int OffenseTotalYards { get; set; }
        public int DefensePassingYards { get; set; }
        public int DefenseRushingYards { get; set; }
        public int DefenseTotalYards { get; set; }
    }
}

File: AngularLovelyjubbly/ClientApp/src/app/shared/models/yardage.ts
import { ITournament } from './tournament';
import { ITeam } from './team';

export interface IYardage {
    yardageId: number;
    tournament: ITournament;
    team: ITeam;
    offensePassingYards: number;
    offenseRushingYards: number;
    offenseTotalYards: number;
    defensePassingYards: number;
    defenseRushingYards: number;
    defenseTotalYards: number;
    netYards: number;
}

export class Yardage implements IYardage {

    constructor(public yardageId: number, public tournament: ITournament, public team: ITeam, public offensePassingYards: number,
        public offenseRushingYards: number, public offenseTotalYards: number, public defensePassingYards: number,
        public defenseRushingYards: number, public defenseTotalYards: number, public netYards: number) {
    }
}
